src/common/model/book/bookParameters.ml

```diff
@@ -3,13 +3,19 @@ open Nes
 type where = Beginning | End | Nowhere
 [@@deriving yojson]
 
+(* FIXME: There are parameters that are really version-, set- or book-specific,
+   but there are others (such as [instruments], [paper_size], etc.) that apply
+   to all. I think we should extract those in a separate type, for instance
+   [GlobalParameters] or [RenderingParameters] or something. *)
```

Comment:
Mind this. This is however a big reorganisation and I do not want to attack it while there are so many open PRs.

Reply:
Looks reasonable, thanks